ci-operator/config/openshift/release/openshift-release-master__nightly-4.16.yaml

```diff
@@ -847,7 +858,7 @@ tests:
     observers:
       enable:
       - observers-resource-watch
-    workflow: network-flow-matrix-single-node-bm
+    workflow: telco5g-network-flow-matrix-single-node-bm
```

Comment:
I see now that `network-flow-matrix-single-node-bm` wokflow is not used at all, does this mean this can be deleted from `step-registry`?

Reply:
I'm going to update it, still checking what's the right flow to use